easybuild/framework/easyblock.py

```diff
@@ -304,10 +304,12 @@ def fetch_patches(self, list_of_patches, extension=False, checksums=None):
             level = None
             if isinstance(patch_entry, (list, tuple)):
                 if not len(patch_entry) == 2:
-                    self.log.error("Unknown patch specification '%s', only two-element lists/tuples are supported!" % patch_entry)
+                    self.log.error("Unknown patch specification '%s', only two-element lists/tuples are supported!",
+                                   str(patch_entry))
                 pf = patch_entry[0]
 
-                if isinstance(patch_entry[1], int):
+                if type(patch_entry[1]) == int:  # int and only int is allowed here, we are parsing a config file, not
```

Comment:
how is this different from the `isinstance` call?

`isinstance` is the right way to check types imho